packages/ksf-pause-subscription/src/KSF/PauseSubscription/Component.purs

```diff
@@ -0,0 +1,233 @@
+    $ DOM.div
+        { className: "clearfix pause-subscription--container"
+        , children:
+            [ Grid.row_
+                [ DOM.div
+                    { className: "col col-11"
+                    , children: [ DOM.h3_ [ DOM.text "Uppehåll på prenumerationen" ] ]
+                    }
+                , DOM.div
+                    { className: "col-1 flex pause-subscription--close-icon"
+                    , children: [ DOM.div { className: "close-icon" } ]
+                    , onClick: handler_ self.props.onCancel
+                    }
+                ]
+            , pauseForm
+            ]
+        }
+  where
+    pauseForm =
+      DOM.form
+          { onSubmit: handler preventDefault (\_ -> submitForm self.state self.props)
+          , children:
+              [ startDayInput
+              , endDayInput
+              , DOM.div
+                  { children: [ submitFormButton ]
+                  , className: "mt2 clearfix"
+                  }
+              ]
+          }
+
+    startDayInput =
+      dateInput
+        self
+        { action: SetStartDate
+        , value: self.state.startDate
+        , minDate: self.state.minStartDate
+        , maxDate: Nothing
+        , disabled: false
+        , label: "Börjar från"
+        }
+
+    endDayInput =
+      dateInput
+        self
+        { action: SetEndDate
+        , value: self.state.endDate
+        , minDate: self.state.minEndDate
+        , maxDate: self.state.maxEndDate
+        , disabled: isNothing self.state.startDate
+        , label: "Skall starta igen"
+        }
+
+    submitFormButton =
+      Grid.columnThird $
+        DOM.button
+          { type: "submit"
+          , children: [ DOM.text "Skicka" ]
+          , className: "button-green"
+          }
+
+type DateInputField =
+  { action   :: Maybe DateTime -> Action
+  , value    :: Maybe DateTime
+  , minDate  :: Maybe DateTime
+  , maxDate  :: Maybe DateTime
+  , disabled :: Boolean
+  , label    :: String
+  }
+
+dateInput :: Self -> DateInputField -> JSX
+dateInput self { action, value, minDate, maxDate, disabled, label } =
+  Grid.row
+    [ Grid.row_ [ DOM.label_ [ DOM.text label ] ]
+    , Grid.row_
+        [ element
+            datePicker
+              { onChange: mkEffectFn1 \pickedDate -> do
+                   send self $ action $ toDateTime =<< toMaybe pickedDate
+              , className: "pause-subscription--date-picker"
+              , value: toNullable $ fromDateTime <$> value
+              , format: "d.M.yyyy"
+              , required: true
+              , minDate: toNullable $ fromDateTime <$> minDate
+              , maxDate: toNullable $ fromDateTime <$> maxDate
+              , disabled
+              , locale: "sv-SV"
+              } ]
+    ]
+    $ Just { extraClasses: [ "mt2" ] }
+
+submitForm :: State -> Props -> Effect Unit
+submitForm { startDate: Just start, endDate: Just end } props@{ userUuid, subsno } = do
+  loginResponse <- Login.loadToken
```

Comment:
🤔 I wonder if we should avoid leaking these details into other components, and have the `Login` component should handle authenticated Persona calls (and at that point we should rename it to `User` component to make it clear).

Another option is to handle all the login-related stuff into a the `Persona` component, but that might be harder

Reply:
Good point, a `User` component makes a lot of sense.